dev/manual_tests/lib/text.dart

```diff
@@ -918,24 +914,22 @@ class _ZalgoState extends State<Zalgo> with SingleTickerProviderStateMixin {
 }
 
 class Painting extends StatefulWidget {
-  const Painting({ Key key, this.seed }) : super(key: key);
+  const Painting({ Key? key, this.seed }) : super(key: key);
 
-  final int seed;
+  final int? seed;
 
   @override
   State<Painting> createState() => _PaintingState();
 }
 
 class _PaintingState extends State<Painting> with SingleTickerProviderStateMixin {
-  String _text;
-  Ticker _ticker;
-  math.Random _random;
+  String? _text;
+  late final Ticker _ticker = createTicker(_update)..start();
+  late math.Random _random = math.Random(widget.seed); // providing a seed is important for reproducibility;
```

Comment:
`late final`?

Reply:
Same comment as `_ZalgoState`, either leave as `late` as you have it, or initialize it:
```dart
  math.Random _random = math.Random();
...
  void initState() {
    super.initState();
    math._random = math.Random(widget.seed); // providing a seed is important for reproducibility;
```